ffv1.md

```diff
@@ -786,6 +786,62 @@ Default values at the decoder initialization phase:
 
 `ConfigurationRecordIsPresent` is set to 0.
 
+## Quantization Table Set
+
+The Quantization Table Sets are stored by storing the number of equal entries -1 of the first half of the table (represented as `len - 1` in the pseudo-code below) using the method described in (#range-non-binary-values). The second half doesn’t need to be stored as it is identical to the first with flipped sign. `scale` and `len_count[ i ][ j ]` are temporary values used for the computing of `context_count[ i ]` and are not used outside Quantization Table Set pseudo-code.
+
+Example:
+
+Table: 0 0 1 1 1 1 2 2 -2 -2 -2 -1 -1 -1 -1 0
+
+Stored values: 1, 3, 1
+
+```c
+pseudo-code                                                   | type
+--------------------------------------------------------------|-----
+QuantizationTableSet( i ) {                                   |
+    scale = 1                                                 |
+    for (j = 0; j < MAX_CONTEXT_INPUTS; j++) {                |
+        QuantizationTable( i, j, scale )                      |
+        scale *= 2 * len_count[ i ][ j ] - 1                  |
+    }                                                         |
+    context_count[ i ] = ceil( scale / 2 )                    |
+}                                                             |
+```
+
+`MAX_CONTEXT_INPUTS` is 5.
+
+```c
+pseudo-code                                                   | type
+--------------------------------------------------------------|-----
+QuantizationTable(i, j, scale) {                              |
+    v = 0                                                     |
+    for (k = 0; k < 128;) {                                   |
+        len - 1                                               | ur
+        for (a = 0; a < len; a++) {                           |
```

Comment:
Is `a` necessary?

Reply:
@retokromer how do you imagine to say in C pseudo-code that the loop has `len` iterations? I don't see how without this `a`.

Reply:
I know that it’s an iteration, but I would say `n` rather than `a` which has usually other meanings.

Reply:
… or possibly even better `l`.